### PR TITLE
feat(*) skip NS based on EnvVar

### PR DESCRIPTION
**Pull request by erickfaustino**
This PR adds the feature to skip namespaces based on `SKIP_NAMESPACES` environment variable and other minor fixes and adjustments

### CODE REVIEW

**Review by souzabrunoftw — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12.7-alpine
+FROM golang:1.12.9-alpine
```
Comment (rasecoiac03) [inline Dockerfile]:
```suggestion
FROM golang:1.13.0-alpine3.10
```

File: cmd/handlers.go [hunk 2]
```diff
@@ -48,29 +47,33 @@ func init() {
...
 
 var (
 	universalDeserializer = serializer.NewCodecFactory(runtime.NewScheme()).UniversalDeserializer()
+	skippedNamespaces     []string
 )
 
 type validationFunc func(*v1beta1.AdmissionRequest) []*utils.Violation
 
 func validateRules(req *v1beta1.AdmissionRequest) []*utils.Violation {
+	skippedNamespaces, _ = utils.GetEnvAsSlice("SKIP_NAMESPACES", ",")
```
Comment (rasecoiac03) [inline cmd/handlers.go]: you could initialise when declaring `skippedNamespaces` instead of every time executing `validateRules`.

**Review by rasecoiac03 — APPROVED** [review]
Very nice. Just suggestions.

File: Dockerfile (hunk 1 above)
Comment (erickfaustino) [reply Dockerfile]: 👍

File: cmd/handlers.go (hunk 2 above)
Comment (erickfaustino) [reply cmd/handlers.go]: Totally agreed.
Thank you for the amazing review as usual

**Review by phspagiari — APPROVED** [review]

**Pull request merged**